### ISSUE TITLE
Alternative to mongodb?

### ISSUE THREAD

**Issue by CardSnap** [milesmcc/LibreNews-Server#23]
Is this project absoultely dependent on mongodb?

To get it running selfhosted with my setup I'd have to build mongodb which takes at least 13GB if the [following](https://github.com/mongodb/mongo/blob/master/docs/building.md) is to be taken as fact.

**Comment by milesmcc** [comment]
Not necessarily...it uses MongoDB to keep track of who has subscribed to desktop push alerts, so if you are able to find another datastore or remove desktop alerts, you should be able to go without Mongo.
